TSPL.docc/LanguageGuide/AccessControl.md

```diff
@@ -44,17 +45,36 @@ A *source file* is a single Swift source code file within a module
 Although it's common to define individual types in separate source files,
 a single source file can contain definitions for multiple types, functions, and so on.
 
+A *package* is a group of modules that you develop as a unit.
+You define the modules that form a package
+as part of configuring the build system you're using,
+not as part of your Swift source code.
+For example, if you use Swift Package Manager to build your code,
+you define a package in your `Package.swift` file
+using APIs from the [PackageDescription][] module;
+if you use Xcode, you specify the package name
+in the Package Access Identifier build setting.
+
+[PackageDescription](https://developer.apple.com/documentation/packagedescription)
```

Comment:
Is this the link that belongs on line 54? Sorry, I'm not understanding how this is organized.

Reply:
Yes, this is the target for the link in the paragraph above.